### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by archiewood**
### Description
Bring in line with most recent messaging

### Checklist
- [ ] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
 
-# Business Intelligence for Modern Data Teams
+# Business Intelligence as Code
 
-Build a polished business intelligence system using only SQL and markdown. 
+Evidence is an open source, code-based alternative to drag-and-drop business intelligence tools.
 
-Evidence offers a modern development workflow for analysts and a polished user experience for decision-makers.
+You write SQL queries alongside markdown files and Evidence compiles them into a data website.
```
Comment (mcrascal) [inline README.md:10]: "data website" isn't a term that I think people will have heard before. I would use "web app", or skip the web part and focus on the actual outputs: reports and analysis? 

e.g. 

"Evidence enables analysts [you] to build reports and analysis using SQL and markdown" or similar.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
 
-![how-it-works](/readme/how-it-works.png)
+![how-it-works](sites/docs/static/img/how-it-works.png)
 
-You can reference the results of those queries directly in your markdown document's text, or you can use them to make charts, graphs, and tables using our built-in component library.
 
-Evidence compiles your markdown documents into a fast and reliable website. It tests your queries and keeps the site up-to-date with your data warehouse.
-
-Evidence supports loops, conditionals, parameterized pages, tests and more. It's easy to get started and you can build data products which are both more sophisticated and dramatically easier to use than what you can build with a typical BI tool.
+Evidence renders a BI website from markdown files:
```
Comment (mcrascal) [inline README.md:17]: BI website is awkward: website or web app will be understood.

**Review by mcrascal — APPROVED** [review]
Nice. 

I flagged a few turns of phrase that feel like contortions. The one other piece of phrasing I don't love is replacing conditionals with if / else. I think conditionals are probably reasonably well understood as a term, but it's not a hill I'd die on.

**Comment by archiewood** [comment]
Going to leave If / Else for now. 

@hughess and I went back and forth on this a bit, and settled on **If / Else** for the docs. 
It's a bit messy as we use **Loops** to describe `{#each}` (we felt each was less well understood), and If / Else to describe `{#if} {:else}`

Happy to revisit. User feedback might help here.

**Pull request merged**